actions/git_clone.sh

```diff
@@ -16,7 +17,8 @@ GITOUTPUT=`$GIT clone -b ${BRANCH} --single-branch $REPO $TARGET`
 
 if [[ $? == 0 ]]
 then
-  echo $TARGET
+  # Use printf so a newline character is not included.
+  printf $TARGET
```

Comment:
I believe `echo -n` should also work.

In any case, I'm also OK with printf :)